design-at-cornell/src/events/Events.tsx

```diff
@@ -0,0 +1,101 @@
+      links: ['google', 'zoom'],
+    },
+    {
+      name: 'Design Seminar Series with Professor XYZ',
+      time: '3:00 - 4:00 PM EST Saturday, May 1, 2021',
+      location: 'Virtual',
+      tags: ['Annual', 'Virtual', 'Computing in the Arts'],
+      description:
+        'As an expert in spatial design studies, Professor XYZ is visiting to give her annual lecture on how designers can better design for and the description goes on and on but this is not a real event so I’m just typing random words until this text space looks filled enough and this seems good so i’ll stop now.',
+      links: ['zoom', 'rsvp', 'facebook'],
+    },
+    {
+      name: 'Design Seminar Series with Professor XYZ',
+      time: '3:00 - 4:00 PM EST Saturday, May 1, 2021',
+      location: 'Virtual',
+      tags: ['Annual', 'Virtual', 'Computing in the Arts'],
+      description:
+        'As an expert in spatial design studies, Professor XYZ is visiting to give her annual lecture on how designers can better design for and the description goes on and on but this is not a real event so I’m just typing random words until this text space looks filled enough and this seems good so i’ll stop now.',
+      links: ['facebook', 'eventLink'],
+    },
+    {
+      name: 'Design Seminar Series with Professor XYZ',
+      time: '3:00 - 4:00 PM EST Saturday, Feb 28, 2021',
+      location: 'Virtual',
+      tags: ['Annual', 'Virtual', 'Computing in the Arts'],
+      description:
+        'As an expert in spatial design studies, Professor XYZ is visiting to give her annual lecture on how designers can better design for and the description goes on and on but this is not a real event so I’m just typing random words until this text space looks filled enough and this seems good so i’ll stop now.',
+      links: ['facebook', 'eventLink'],
+    },
+    {
+      name: 'Design Seminar Series with Professor XYZ',
+      time: '3:00 - 4:00 PM EST Saturday, Feb 28, 2019',
+      location: 'Virtual',
+      tags: ['Annual', 'Virtual', 'Computing in the Arts'],
+      description:
+        'As an expert in spatial design studies, Professor XYZ is visiting to give her annual lecture on how designers can better design for and the description goes on and on but this is not a real event so I’m just typing random words until this text space looks filled enough and this seems good so i’ll stop now.',
+      links: ['facebook', 'eventLink'],
+    },
+  ]);
+
+  const date = new Date();
+
+  const [year, setYear] = useState(date.getFullYear());
+  const [month, setMonth] = useState(date.toLocaleString('default', { month: 'long' }));
+
+  const leftClickHandler = () => setYear(year - 1);
+  const rightClickHandler = () => setYear(year + 1);
+  const monthClickHandler = (m: string) => setMonth(m);
+
+  const filteredEvents = events.filter(
+    (event) => event.time.includes(month.slice(0, 3)) && event.time.includes(year.toString())
```

Comment:
If separated, this filtering wouldn't need to go through the entire string with `.includes()` to find the month (probably a shorter string with just the date or even if time is separated into month, day, year, it could be directly compared), and same with filtering by year.